helix-lsp/src/lib.rs

```diff
@@ -312,7 +313,22 @@ impl Registry {
                 Entry::Vacant(entry) => {
                     // initialize a new client
                     let id = self.counter.fetch_add(1, Ordering::Relaxed);
-                    let (mut client, incoming) = Client::start(&config.command, &config.args, id)?;
+                    let (mut client, incoming) = Client::start(
+                        &config.command,
+                        &config.args,
+                        match load_lsp_config_file(&language_config.language_id) {
+                            Ok(c) => c,
+                            Err(e) => {
+                                log::info!(
+                                    "Loading lsp config file for {} failed => {}",
+                                    language_config.language_id,
+                                    e
+                                );
+                                None
+                            }
```

Comment:
I don't think we want to show that it failed when the custom config isn't there.

Reply:
Yeah agreed